jbpm-services/jbpm-executor/src/main/java/org/jbpm/executor/commands/LogCleanupCommand.java

```diff
@@ -106,6 +106,7 @@ protected int readInt(Map<String, Object> params, String propName, int defaultVa
 
     @Override
     public ExecutionResults execute(CommandContext ctx) throws Exception {
+        long startTime = Instant.now().toEpochMilli();
```

Comment:
instead of this I would use the ctx to get the running time. AbstractAvailableJobsExecutor::executeGivenJob during the ctx init.
https://github.com/kiegroup/jbpm/blob/main/jbpm-services/jbpm-executor/src/main/java/org/jbpm/executor/impl/AbstractAvailableJobsExecutor.java#L129
There is the proper time in there.

RequestInfo.getTime you have there the time scheduled not something computed, so it would be more accurate strictly speaking.

Reply:
great idea, thanks